src/style/style_layer/line_style_layer.js

```diff
@@ -44,7 +44,7 @@ class LineStyleLayer extends StyleLayer {
     _unevaluatedLayout: Layout<LayoutProps>;
     layout: PossiblyEvaluated<LayoutProps>;
 
-    gradientTextureInvalidated: boolean;
+    gradientGeneration: number;
```

Comment:
nit: The `ImageAtlas` and `ImageManager` use `version` to represent multiple generations of the same resource. Should we be consistent with that terminology?

Reply:
Makes sense! Updated.